packages/synapse-react-client/src/components/BasePortalCard/ColorfulPortalCardWithChips/ColorfulPortalCardWithChips.tsx

```diff
@@ -108,7 +114,15 @@ const ColorfulPortalCardWithChips = ({
             variant="smallText1"
             className={styles.ColorfulPortalCardWithChips__description}
           >
-            {description}
+            <SynapseCardLabel
```

Comment:
Can you just use the Markdown component?  Or are you using SynapseCardLabel for some other reason?